### PR TITLE
Remove CI projects from main repo

### PR DESCRIPTION
**Pull request by hartez**
### Description of Change ###

Removes the Xamarin.Forms.Loader, Xamarin.Forms.UITest.TestCloud, and Xamarin.Forms.UITest.Validation projects from the main repo. These projects are all specifically related to the Xamarin.Forms continuous integration pipeline, and thus don't belong in the main public repository.

### API Changes ###

None.

### Behavioral Changes ###

None.

### CODE REVIEW

**Comment by rmarinho** [comment]
👍

File: PagesGallery/PagesGallery/PagesGallery.csproj [hunk 1]
```diff
@@ -71,6 +71,11 @@
       <Project>{9db2f292-8034-4e06-89ad-98bbda4306b9}</Project>
       <Name>Xamarin.Forms.Xaml</Name>
     </ProjectReference>
+	  <ProjectReference Include="..\..\Xamarin.Forms.Build.Tasks\Xamarin.Forms.Build.Tasks.csproj">
+		  <Project>{96D89208-4EB9-4451-BE73-8A9DF3D9D7B7}</Project>
+		  <Name>Xamarin.Forms.Build.Tasks</Name>
+		  <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+	  </ProjectReference>
```
Comment (StephaneDelcroix) [inline PagesGallery/PagesGallery/PagesGallery.csproj]: why this ?

Comment (rmarinho) [inline PagesGallery/PagesGallery/PagesGallery.csproj]: We got this error on our build server... 

 error : Error initializing task FixedCreateCSharpManifestResourceName: Could not load file or assembly '/Applications/buildAgent_c/work/824090adae967c0/.nuspec/Xamarin.Forms.Build.Tasks.dll' or one of its dependencies
[17:38:06][Step 3/6]    		Build FAILED.

adding the reference fixes it

Comment (rmarinho) [inline PagesGallery/PagesGallery/PagesGallery.csproj]: Oh and this was when compiling in Pages csproj...

Comment (StephaneDelcroix) [inline PagesGallery/PagesGallery/PagesGallery.csproj]: does not seem the right fix for this issue at all

**Comment by jassmith** [comment]
:+1:

**Comment by kingces95** [comment]
Where will these live now?

**Comment by rmarinho** [comment]
CI stuff will live on https://github.com/xamarin/Xamarin.Forms.Keys for now..

File: PagesGallery/PagesGallery/PagesGallery.csproj (hunk 1 above)
Comment (hartez) [inline PagesGallery/PagesGallery/PagesGallery.csproj]: Removed; we'll figure out how to fix this on another branch.

**Pull request merged**